_targets.R

```diff
@@ -7,10 +7,36 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
```

Comment:
redundant - would love suggestions on how to streamline, as I expressed in my general comment